test/regression/RegressionTest.cc

```diff
@@ -38,21 +38,25 @@ void RegressionTest::run(const char* source, const char* triple,
   memory_ = std::make_shared<simeng::memory::SimpleMem>(memorySize);
 
   // Initialise a SimOS object.
-  simeng::OS::SimOS OS = simeng::OS::SimOS(DEFAULT_STR, {}, memory_, true);
+  simeng::OS::SimOS OS = simeng::OS::SimOS(DEFAULT_STR, {}, memory_);
 
   // Create a Process from the assembled code block.
   // Memory allocation for process images also takes place
   // during Process creation. The Elf binary is parsed
```

Comment:
I know this change isn't from your PR but the comment is wrong.  Please remove the sentence starting from "The Elf binary is parsed..."

Reply:
Done